packages/app/src/webapp/views/sidecar.ts

```diff
@@ -117,8 +117,11 @@ export const hide = (tab: Tab, clearSelectionToo = false) => {
   const replView = tab.querySelector('.repl')
   replView.classList.remove('sidecar-visible')
 
-  // we just hid the sidecar. make sure the current prompt is active for text input
-  // cli.getCurrentPrompt().focus()
+  // we just hide the sidecar. make sure the current prompt is active for text input
+  const promptToFocus = getCurrentPrompt(tab)
+  if (promptToFocus) {
+    promptToFocus.focus()
+  }
```

Comment:
My PR includes this change because without `repl` grabbing focus, the tab cycle is wrong as below:

Step 1. `hit Tab` to `the sidecar close button` and `hit Enter`
Should see: the focus is on the close button even if it’s hidden
Step 2: continue to `hit Tab`
Should see: the focus is on the `first top tab` rather than the `repl input`
Step 3:  To focus on `repl input`, we need to `hit shift+Tab` to go backward or using mouse